### PR TITLE
Fix/sniff soc home page test it

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java [hunk 1]
```diff
@@ -0,0 +1,885 @@
...
+   * (icon) Input Data: Expected Outcome: - Select File Dialog shows up
+   */
+
+  /*
+   * Step number: 2 Step Name: - Select document Step Description: - Browse the
+   * Select File Dialog to choose a document - Click Select Input Data: Expected
+   * Outcome: - A breadcrumb is displaying the current position of the user in the
+   * browsed drive. - the name of the file should be displayed under the activity
+   * input field.
+   */
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java]: Comments should be merged (one comment block per method) for a better readability.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java [hunk 2]
```diff
@@ -0,0 +1,885 @@
...
+  @Test
+  public void test09_LoadPreviousActivityPageAutomatically() {
+    info("Test 9: Load previous activity page automatically");
+    String textDesfirst = "textDesfirst" + getRandomNumber();
+    activityStream.addActivity(textDesfirst, "");
+    refresh();
+    for (int i = 0; i <= 15; i++) {
+      String act = "act" + getRandomNumber();
+      activityStream.addActivity(act, "");
+      refresh();
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java]: Do we really need to refresh at each iteration ?

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java [hunk 3]
```diff
@@ -0,0 +1,885 @@
...
+
+    homePagePlatform.goToHomePage();
+    activityStream.addActivity(name, "");
+    $(byText(name)).parent()
+                   .parent()
+                   .parent()
+                   .parent()
+                   .parent()
+                   .parent()
+                   .parent()
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java]: This looks very fragile. No way to use an id or a class somewhere in the DOM ?

File: social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java [hunk 4]
```diff
@@ -20,6 +22,9 @@
 import org.exoplatform.platform.qa.ui.selenium.platform.ActivityStream;
 import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
 import org.exoplatform.platform.qa.ui.selenium.platform.NavigationToolbar;
+import org.openqa.selenium.By;
+
+import java.awt.*;
```
Comment (thomasdelhomenie) [inline social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java]: All these new imports are useless.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java (hunk 2 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java]: refresh removed

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java (hunk 1 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java]: done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java (hunk 3 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java]: $(byText(name)).parent() .parent() .parent() .parent().parent().parent().parent().find(ELEMENT_PUBLICATION_FIRSTPOST_AUTHORAVATAR).should(Condition.exist);

File: social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java (hunk 4 above)
Comment (ilyeskallala) [reply social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java]: code formatted

**Pull request merged**